src/PowerShellEditorServices.Protocol/Server/DebugAdapter.cs

```diff
@@ -469,7 +471,24 @@ protected void Stop()
             // will block the debug adapter initialization process.  The
             // InitializedEvent will be sent as soon as the RunspaceChanged
             // event gets fired with the attached runspace.
-            int runspaceId = attachParams.RunspaceId > 0 ? attachParams.RunspaceId : 1;
+
+            int runspaceId = 1;
```

Comment:
nit: we use var when it's obvious

```suggestion
            var runspaceId = 1;
```